src/core/lwt.mli

```diff
@@ -1868,6 +1868,9 @@ val pause : unit -> unit t
     Putting the rest of your computation into a callback of [Lwt.pause ()]
     creates a “yield” that gives other callbacks a chance to run first.
 
+    To wait for one complete iteration of the main loop you need to wait for
+    [Lwt.pause () >>= Lwt.pause]
+
```

Comment:
As the change of code makes obvious, this is a property of the scheduler, not the `pause` function. I'd move the documentation there because:
- this might be different on different platforms (e.g., js-of-ocaml)
- the notion of main loop is only mentioned in this file as part of a comment about `Lwt_main.run` in the "Execution model" section which is intended more as an introduction / presentation of general concepts

Reply:
I added the documentation to `Lwt.main`.